2.hello-kubernetes/node/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "node_server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
```

Comment:
We don't need to check this in.

Reply:
We need this :) otherwise, npm install doesn't work